recipes/soem/meta.yaml

```diff
@@ -0,0 +1,49 @@
+{% set name = "soem" %}
+{% set version = "1.4.0" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  - url: https://github.com/OpenEtherCATsociety/{{ name }}/archive/v{{ version }}.tar.gz
+    sha256: e58e1e0c216dade84a76d2c10ab34d2fa156ca639eac00a8081be6e88911d86e
+
+build:
+  number: 0
+  skip: true  # [osx]
+  run_exports:
+    - {{ pin_subpackage(name, max_pin='x.x') }}
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}              # [not win]
+    - {{ compiler('c') }}                # [not win]
+    - vs2017_win-64                      # [win64]
+    - vs2017_win-32                      # [win32]
+    - ninja
+    - cmake
+  host:
+    - pthread-stubs                      # [not win]
+  run:
+    - pthread-stubs                      # [not win]
+
+test:
+  commands:
+    - test -f ${PREFIX}/include/soem/ethercat.h  # [not win]
+    - test -f ${PREFIX}/bin/slaveinfo  # [not win]
+    - test -f ${PREFIX}/lib/libsoem.a  # [not win]
```

Comment:
Can you build shared libraries?

Reply:
I don't know how far the mileage would be, but let me try. :)